src/main/java/com/example/doodle_clone/repo/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.doodle_clone.repo;
+
+import com.example.doodle_clone.models.user.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, String> {
+    //@Query("SELECT u FROM user u WHERE u.email = ?1")
```

Comment:
leave todo if still relevant (I suppose not)

Reply:
fix